codersdk/workspaceagents.go

```diff
@@ -439,6 +439,10 @@ type WorkspaceAgentContainer struct {
 	// Volumes is a map of "things" mounted into the container. Again, this
 	// is somewhat implementation-dependent.
 	Volumes map[string]string `json:"volumes"`
+	// DevcontainerDirty is true if the devcontainer configuration has changed
+	// since the container was created. This is used to determine if the
+	// container needs to be rebuilt.
+	DevcontainerDirty bool `json:"devcontainer_dirty"`
```

Comment:
Suggestion: reduce stuttering in name
```suggestion
	Dirty bool `json:"dirty"`
```

Reply:
This is the container structure, so I wouldn’t consider this stutter. A container being dirty isn’t very informative. I’m sure we want to consolidate/simplify the API a bit down the line so this could change later.